packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -63,7 +63,9 @@
 
   .avatar > .image {
     border-radius: 50%;
+    font-family: 'object-fit: cover;';
```

Comment:
Let's add a comment that this is used to provide settings to the object-fit polyfill